packages/template-retail-react-app/package.json

```diff
@@ -96,7 +96,7 @@
     },
     {
       "path": "build/vendor.js",
-      "maxSize": "352 kB"
+      "maxSize": "353 kB"
```

Comment:
FYI had to increase bundle size of `template-retail-react-app`. I'm guessing this is because of the changes i made to `pwa-kit-react-sdk`